### PR TITLE
Tweaks to Bot Template

### PR DESCRIPTION
**Pull request by JBraunsmaJr**
1. Removed Net5 framework selection/support as Net6 is now released. 
2. Removed installation documentation, when we have a ready to use template we can add it back
3. Adjusted file structure to utilize file-scoped namespace
4. Updated Nuspec name
5. Updated Git URL references

### CODE REVIEW

File: src/DisCatSharpTemplates.nuspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
-    <id>DisCatSharp.Hosting.ProjectTemplates</id>
+    <id>DisCatSharp.ProjectTemplates</id>
     <version>1.0.01</version>
```
Comment (Lulalaby) [inline src/DisCatSharpTemplates.nuspec]:
```suggestion
    <version>1.0.2</version>
```

Comment (JBraunsmaJr) [reply src/DisCatSharpTemplates.nuspec]: fixed

**Comment by JBraunsmaJr** [comment]
Add parameter for bot token for the bot template

image

**Comment by JBraunsmaJr** [comment]
Given the scenario multiple bot projects exist in a solution:

Web Host Template has an added task which copies all JSON files within the solution into the output directory.
This ensures all JSON files required for the bot(s) to run are included

**Comment by Lulalaby** [comment]
❤️

**Comment by JBraunsmaJr** [comment]
As noted in discord

Added solution template which gets updated when you run build-nuget.ps1

Updated individual project templates naming conventions to make it easier to aggregate

Added classifications to make template discovery easier

**We will have to update this project to point to the next stable release**

**Comment by Lulalaby** [comment]
Okidoki

**Comment by Lulalaby** [comment]
Bumped to 9.8.4 already for the release

**Comment by JBraunsmaJr** [comment]
Added git-tagging support so we can tag via github which will represent the version number that'll get produced via build-nuget.ps1

It is important to note that the tags can only contain digits and decimal/periods 

9.8.4 is valid
9.8.4-rc1, for instance is not

**Comment by Lulalaby** [comment]
That's nice!

**Comment by JBraunsmaJr** [comment]
Apparently I don't have the access I used to have on this repo, but regardless I don't have the info for your nuget stuff anyhow

IDK what you want the keyname to be but inside of the SECRETS portion of the repo settings an API key needs to be set, it gets loaded into the workflow via ```${{ secrets.NUGET_API_KEY }}```

It's untested but in theory this should work as the other repo did something similar. Only thing I'm worried about is if the powershell stuff works first go through

**Comment by Lulalaby** [comment]
Api key is set org wide

**Comment by Lulalaby** [comment]
image
This should work?

**Comment by JBraunsmaJr** [comment]
Didn't know there was an org one.  That will work though.

**Comment by Lulalaby** [comment]
I keep most secrets org-wide, because we need them over and over again 
:doggolul:

**Review by Lulalaby — DISMISSED** [review]

**Pull request merged**